network-api/networkapi/tests/tests.py

```diff
@@ -407,7 +407,7 @@ def test_general_page_export_fields(self):
 
 class TestCreateWagtailImageUtility(TestCase):
     def setUp(self):
-        self.image_path = abspath(join(dirname(__file__), "../media/images/placeholders/products/teddy.jpg"))
+        self.image_path = abspath(join(dirname(__file__), "../../media/images/placeholders/products/teddy.jpg"))
```

Comment:
Since I moved this file to the newly created /tests/ directory, I had to update this path.